test/Microsoft.AspNetCore.Server.Kestrel.Performance/RequestParsing.cs

```diff
@@ -23,6 +23,10 @@ public class RequestParsing
 
         private const string plaintextRequest = "GET /plaintext HTTP/1.1\r\nHost: www.example.com\r\n\r\n";
 
+        private const string plaintextTechEmpower = "GET /plaintext HTTP/1.1\r\n" +
+            "Host: localhost\r\n" +
+            "Accept: text/plain,text/html;q=0.9,application/xhtml+xml;q=0.9,application/xml;q=0.8,*/*;q=0.7\r\n\r\n";
```

Comment:
@KrzysztofCwalina sorry for randomizing, but please add `Connection: keep-alive` back. TechEmpower sends it. cc @mikeharder 

In E2E we benchmark with the `Connection` header: https://github.com/aspnet/benchmarks/blob/70427d48dafecaaa8429fe0ff5fc98a7435c9e5b/src/BenchmarksDriver/Program.cs#L28